src/applications/discharge-wizard/components/AirForcePortalLink.jsx

```diff
@@ -4,7 +4,7 @@ const AirForcePortalLink = () => (
   <section className="vads-u-margin-bottom--9">
     <a
       className="vads-c-action-link--green"
-      href="https://afrba-portal.cce.af.mil/#portal"
+      href="https://afrba-portal.cce.af.mil/"
     >
       Get the form on the Air Force portal
```

Comment:
Should this link say `Get the form on the Air Force Review Boards Agency Website and Portal.`?